starter-code/css/main.css

```diff
@@ -6,3 +6,110 @@ Green: #00B172
 White: #FFF
 
 */
+body {
+    font-family: Helvetica Neue, Sans-Serif;
+}
+
+  /* nav bar */
+
+.nav-bar {
+    position: fixed;
+    top: 0;
+    width: 99%;
+    height: 83px;
+    background-color: white;
+}
+.nav-bar .links ul {
+       float: right;
+       list-style: none;
+       display: flex;
+        margin-top: 35px;
+        margin-right: -1%;
+        color: #1A1A1A;
+}
+.nav-bar .links ul li{
+    padding-right: 50px;
+}
+.logo img {
+    float: left;
+    height: 35px;
+    width: 110px;
+    margin-top: 30px;
+    margin-left: 3%;
+}
+.clear {
+      clear: both;
+}
+
+/* everyone */
+
+.everyone {
+    margin-top: 83px;
+    background-image: url(../images/landing.jpg);
+    background-size: 100% 100%;
+    height: 600px;
+    color: white;
+}
+.everyone h1 {
+    text-align: center;
+    font-size: 4.9em;
+    padding-top: 130px;
+}
+.everyone p {
+    width: 52%;
+    margin-left: 24%;
+    text-align: center;
+    font-size: 2em;
+}
+
+/* whats-on */
+
+.whats-on {
+    margin: 100px 20px 230px 0px;
+}
+
+.whats-on h2 {
```

Comment:
I will admit that  I think this part was hard to get right 😁, you would actually not use a bottom border green ....

You could have done something like this for the HTML:

```html
                    <div class="fancy-border-green">
                        <h2>What’s on Spotify?</h2>
                    </div>
```

and this for the CSS:
```css
.fancy-border h2 {
    text-align: center;
}
.fancy-border-green {
	width: 360px;
	position: relative;
	margin: 0 auto;

}
.fancy-border-green:after {
    border-bottom: 4px solid #1db954;
    content: '';
    height: 1px;
    position: absolute;
    left: 0;
    bottom: -5px;
    width: 100%;
}
```